README.md

```diff
@@ -36,11 +51,20 @@ Additionally, the REST API exposes the same core functionality as the Web UI and
 
 ### Component Plugin Architecture
 
+The OpenMPF Plugin Architecture provides the ability to seamlessly integrate **detection** , **tracking** , and **classification** algorithms in both C++ and Java.
```

Comment:
There is a space between detection and tracking and their commas. 
![screen shot 2017-10-10 at 11 02 20 am](https://user-images.githubusercontent.com/5748179/31393787-88448dd2-adaa-11e7-9e6f-13327b4d6809.png)